CHANGES.rst

```diff
@@ -49,6 +49,8 @@ gaia
 
 - The method ``get_datalinks`` can be used with the new parameter linking_parameter. It completes PR #2859. [#2936]
 
+- Update the docstring for the returned value of the method ``cross_match``. [#2937]
```

Comment:
no need for changelog for docs related changes. (the need for documenting changelog howto has already raised, so I'll try to write it up sometimes soon.)

https://github.com/astropy/pyvo/issues/516